.asdf

```diff
@@ -1,37 +0,0 @@
-### Examples: ###
-# Employ asdf for go, ruby, and python used by this project only:
```

Comment:
no longer needed - `act` will containerize these dependencies if running linters locally is desired, and between intern and our test action we cover enough golang version ranges that we don't need to be overly opinionated about any version in the range